src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs

```diff
@@ -907,23 +907,23 @@ internal bool IsSerializableContent(string propertyName, object parent)
 			bool	serializable = true;
 			if(_serializableContent.Length > 0 || _nonSerializableContent.Length > 0)
 			{
-				int		serialzableClassFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-				int		serialzablePropertyFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
                 string ownerClassName = GetObjectName(parent);
 
-				// Check if property in this class is part of the serializable content
-				serializable = IsPropertyInList(GetSerializableContentList(), ownerClassName, propertyName, out serialzableClassFitType, out serialzablePropertyFitType);
+                // Check if property in this class is part of the serializable content
+                serializable = IsPropertyInList(GetSerializableContentList(), ownerClassName, propertyName, out int serialzableClassFitType, out int serialzablePropertyFitType);
 
 				// Check if property in this class is part of the NON serializable content
 				if(serializable)
 				{
-					int		nonSerialzableClassFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-					int		nonSerialzablePropertyFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-					bool	nonSerializable = IsPropertyInList(GetNonSerializableContentList(), ownerClassName, propertyName, out nonSerialzableClassFitType, out nonSerialzablePropertyFitType);
+                    // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                    // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
```

Comment:
Lost the meaning of the comments.

Reply:
This comments was for:
`nonSerialzableClassFitType`
and
`nonSerialzablePropertyFitType`